### PR TITLE
whitelist spark_price_transparency

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist spark_price_transparency

### Linked issues
Progresses #1901 

### Functionality 
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -1977,6 +1977,228 @@
...
+    "spark_price_transparency.pt_schema": [
+      {
+        "code": "dbfs-usage",
+        "message": "Deprecated file system path: dbfs:/tmp/pt/_checkpoint"
+      }
+    ],
+    "spark_price_transparency.pt_stage": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self._spark.sql(f'CREATE DATABASE IF NOT EXISTS {self.stage_db_name}')' because its value cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2116]: noise

File: src/databricks/labs/ucx/source_code/known.json [hunk 2]
```diff
@@ -1977,6 +1977,228 @@
...
+      }
+    ],
+    "spark_price_transparency.pt_table": [
+      {
+        "code": "table-migrate",
+        "message": "SQL query is not supported yet: DELETE FROM  WHERE 1=1"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2122]: noise. please change the error code for that

File: src/databricks/labs/ucx/source_code/known.json [hunk 3]
```diff
@@ -1977,6 +1977,228 @@
...
+      {
+        "code": "table-migrate",
+        "message": "The default format changed in Databricks Runtime 8.0, from Parquet to Delta"
+      }
+    ],
+    "spark_price_transparency.pt_types": [],
+    "spark_price_transparency.table": [
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate 'self.spark.table(self.wh_tbl_name)' because its table name argument cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2137]: noise. please change the error code for that

File: src/databricks/labs/ucx/source_code/known.json [hunk 4]
```diff
@@ -1977,6 +1977,228 @@
...
+      },
+      {
+        "code": "table-migrate",
+        "message": "Can't migrate table_name argument in 'self.spark.sql(f'DELETE FROM {self.wh_tbl_name} WHERE 1=1')' because its value cannot be computed"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2141]: noise. please change the error code for that

File: src/databricks/labs/ucx/source_code/known.json [hunk 5]
```diff
@@ -1977,6 +1977,228 @@
...
+      },
+      {
+        "code": "table-migrate",
+        "message": "The default format changed in Databricks Runtime 8.0, from Parquet to Delta"
+      }
+    ],
+    "spark_price_transparency.table_analytic": [
+      {
+        "code": "table-migrate",
+        "message": "SQL query is not supported yet: DELETE FROM  WHERE 1=1"
```
Comment (nfx) [inline src/databricks/labs/ucx/source_code/known.json:2155]: noise. please change the error code for that

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: removed, submitted #https://github.com/databrickslabs/ucx/pull/2018

File: src/databricks/labs/ucx/source_code/known.json (hunk 2 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: removed, submitted #https://github.com/databrickslabs/ucx/pull/2018

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: done

File: src/databricks/labs/ucx/source_code/known.json (hunk 4 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: done

File: src/databricks/labs/ucx/source_code/known.json (hunk 5 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json]: done

**Review by nfx — APPROVED** [review]
lgtm

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**